services/app-api/utils/validation/schemaMap.test.ts

```diff
@@ -90,6 +102,11 @@ describe("Schemas", () => {
     testSchema(number(), badNumberTestCases, false);
   });
 
+  test("Evaluate Number Schema using number positive scheme", () => {
+    testSchema(numberPositive(), goodPositiveNumberTestCases, true);
+    testSchema(numberPositive(), badNumberTestCases, false);
```

Comment:
Request: Could you add a test case for zero, and one for a negative number? For this schema and the other two.

Reply:
New test added.